### ISSUE TITLE
Support for Venu 2 (Plus)

### ISSUE THREAD

**Issue by konstantinj** [cedric-dufour/connectiq-widget-totp#2]
Hi, is support for Venu series possible?

**Comment by exo-cedric** [comment]
No. It cannot work without [TextPicker](https://developer.garmin.com/connect-iq/api-docs/Toybox/WatchUi/TextPicker.html) (which is not available on the Touch devices; like Venu 2) or significant re-factoring
